src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
Need to check for explicit false here. We don't want the case where a no-op callback returns `undefined` and so we throw away the payload.